packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/common_services/users_service.dart

```diff
@@ -0,0 +1,64 @@
+import '../models/user_model.dart';
+import '../models/user_with_auth_token_model.dart';
+import '../repositories/url_launcher_repository.dart';
+import '../repositories/users_repository.dart';
+
+/// Fake deep link for successful email confirmation.
+/// Used for demo purposes, should be removed in a real app
+const mockEmailDeepLinkError =
+    '{{project_name}}://{{project_name}}/onboarding/email-confirmed/00000000';
+
+/// Fake deep link for unsuccessful email confirmation.
+/// Used for demo purposes, should be removed in a real app
+const mockEmailDeepLinkSuccess =
+    '{{project_name}}://{{project_name}}/onboarding/email-confirmed/11111111';
+
+class UsersService {
+  UsersService(
```

Comment:
Do you think that the `OnboardingService` fits better as a class name ?

Reply:
Renamed class `UsersService` to `OnboardingService` in all relevant places.